docs/book/src/quick-start.md

```diff
@@ -65,7 +65,7 @@ kubebuilder init --domain my.domain --repo my.domain/guestbook
 If your project is initialized within [`GOPATH`][GOPATH-golang-docs], the implicitly called `go mod init` will interpolate the module path for you.
 Otherwise `--repo=<module path>` must be set.
 
-Read the [Go modules blogpost][go-modules-blogpost] if unfamiliar with the module system.
+Read the [Go modules blogpost](https://blog.golang.org/using-go-modules) if unfamiliar with the module system.
```

Comment:
I think we can change L226
https://github.com/kubernetes-sigs/kubebuilder/blob/b6ecf5ef413852d8ca2a7b0281c54e316dbb5b29/docs/book/src/quick-start.md#L226
```
[go-module-blogpost]: https://blog.golang.org/using-go-modules
```

Reply:
Good idea! You're right! Thanks